### PR TITLE
Add more social crimes

### PR DESCRIPTION
**Pull request by TolstoyDotCom**
Several were missing. Please consult the leaders of Russia, China, Turkey, Saudi Arabia, Iran, etc for even more.

### CODE REVIEW

**Comment by bendov3r** [comment]
There are only two genders

**Comment by abetwothree** [comment]
Looks good to me! Ship to prod!

**Review by acenoor — COMMENTED** [review]
@TolstoyDotCom : what's a social crime, can you add more context in the PR description, thanks

**Comment by TolstoyDotCom** [comment]
Thank you for asking. A "social crime" is a crime against society. Something like a robbery is a crime against an individual, but something like denying the election results is a crime against society as a whole. Denying the election results (does anyone doubt President Putin has won all of his elections?) is correctly punished by Twitter but more needs to be done. For instance, some today deny the results of the Norman Conquest, and Twitter should punish them too.

For another example, Mahsa Amini didn't mug anyone, but she did commit a crime against society by improperly wearing the hijab. [Twitter, to their great credit, censors half the replies to Ayatollah Khamenei](https://tolstoy.com/twittercensorship/), but they must go further and punish all who support shocking crimes against society like improper hijab wearing or even not wearing the hijab at all.

Hopefully someone from Twitter's Trust and Safety team will explain why they only punish some social crimes but not all.

File: visibilitylib/src/main/scala/com/twitter/visibility/models/SpaceSafetyLabelType.scala [hunk 1]
```diff
@@ -33,15 +33,84 @@ object SpaceSafetyLabelType extends SafetyLabelType {
     s.SpaceSafetyLabelType.MedicalMisinfo -> MedicalMisinfo,
     s.SpaceSafetyLabelType.GenericMisinfo -> GenericMisinfo,
     s.SpaceSafetyLabelType.DmcaWithheld -> DmcaWithheld,
+    s.SpaceSafetyLabelType.AntiChineseMisinfo -> AntiChineseMisinfo,
+    s.SpaceSafetyLabelType.MisnamingTaiwan -> MisnamingTaiwan,
```
Comment (randiaz95) [inline visibilitylib/src/main/scala/com/twitter/visibility/models/SpaceSafetyLabelType.scala:37]: LOL

**Pull request closed without merging**